qa/tasks/deepsea.py

```diff
@@ -868,6 +868,71 @@ def begin(self):
     def teardown(self):
         pass
 
+class State(DeepSea):
```

Comment:
Could you please describe the class purpose and usage examples.

Reply:
done